internal/cloud/user.go

```diff
@@ -0,0 +1,53 @@
+package cloud
+
+import (
+	"bytes"
+	"os/exec"
+	"regexp"
+
+	"github.com/pkg/errors"
+	"go.jetpack.io/devbox/internal/debug"
+)
+
+var githubSSHRegexp = regexp.MustCompile("Hi (.+)! You've successfully authenticated, " +
+	"but GitHub does not provide shell access")
```

Comment:
Does this break if github changes its error message lol. Should we shorten this and just match with `Hi (.+)!`?

Reply:
It'll fallback to having the user prompt, so stuff should still be workable, but less frictionless.. 

I guess I was inclined to leave the longer message, so that we intentionally update this if GitHub ever changes their error message (not likely to be a frequent occurence).